### PR TITLE
Webpack devserver

### PR DESCRIPTION
**Pull request by EmanekaT**
### Summary
Reset the yarn run devserver command to use yarn-run-all, such that webpack-devserver and django runserver run seperately. When webpack-devserver terminates, django runserver terminates as well.

### References
 * Issue #2391
 * https://www.npmjs.com/package/yarn-run-all

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -123,10 +124,11 @@
     "webpack-bundle-size-analyzer": "^2.0.2",
     "webpack-bundle-tracker": "https://github.com/learningequality/webpack-bundle-tracker",
     "webpack-dev-middleware": "^1.9.0",
-    "webpack-dev-server": "^2.3.0",
-    "webpack-hot-middleware": "^2.12.1",
-    "webpack-merge": "^4.1.0",
-    "webpack-rtl-plugin": "^1.6.0"
+    "webpack-dev-server": "2.3.0",
+    "webpack-hot-middleware": "2.12.1",
+    "webpack-merge": "^2.5.0",
```
Comment (rtibbles) [inline package.json]: Can you revert the change to the `webpack-merge` version here? It should be `^4.1.0`. Will need to regenerate the yarn.lock file when you do this also.

**Comment by rtibbles** [comment]
Well, I think the instructions would technically still work, but I would quite like to deprecate the devserver command, and not invoke webpack from inside django.

File: package.json [hunk 2]
```diff
@@ -12,14 +12,15 @@
     "build": "webpack --config ./frontend_build/src/webpack.config.prod.js",
     "test-build": "yarn run lint-js && webpack --config ./frontend_build/src/webpack.config.prod.js --bail",
     "watch": "node ./frontend_build/src/webpackdevserver.js",
+    "django-devserver": "kolibri --debug manage devserver --settings=kolibri.deployment.default.settings.dev ",
+    "devserver": "./node_modules/.bin/npm-run-all --parallel django-devserver watch",
```
Comment (jonboiser) [inline package.json:16]: you can omit the path to `npm-run-all`. in the context of package.json scripts, `.bin` is considered as part of the "path"

Comment (rtibbles) [reply package.json:16]: I have seen cases where this has not worked consistently, which is why the fuller path is here.
